### DISCUSSION TITLE
Add watch full text of prompts in the main window

### DISCUSSION THREAD

**Discussion by Dylanmengzhou** [intellibar/main#29] [Ideas]
I wish I could see the previous prompts before(full text), also could copy the previous prompt

**Comment by astoilkov** [comment]
I'm not sure I understand this. Can you explain what you are imagining in more detail? Thanks!

**Reply by Dylanmengzhou** [reply]
Hi, I just wish I could see the full content of the previous information, not just a fragment.
<img width="672" alt="iShot_2023-11-22_15 56 24" src="https://github.com/intellibar/main/assets/61518219/ab04b3fb-4afe-4e48-be69-5b66e19f114d">

**Reply by astoilkov** [reply]
Aha, got it. Thanks. Is it going to work for you if you see the entire prompt on hover (in a tooltip)?

**Reply by Dylanmengzhou** [reply]
that will work! If I could turn on that option, It would be really convenient.

**Reply by astoilkov** [reply]
Awesome. It's done. It will be available in the next release.
